CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to hdx-signals
+
+This document outlines how to contribute to the HDX Signals code repository.
+
+## Simple text changes
+
+You can fix typos, spelling mistakes, or grammatical errors in the documentation directly using the GitHub web interface.
+
+## Bigger changes
+
+If you want to make a bigger change, it's a good idea to first file an issue and make sure someone from the team agrees that it’s needed. 
+If you’ve found a bug, please file an issue that illustrates the bug with a minimal 
+[reprex](https://www.tidyverse.org/help/#reprex) (this will also help you write a unit test, if needed).
+See the Tidyverse guide on [how to create a great issue](https://code-review.tidyverse.org/issues/) for more advice.
+
+### Pull request process
+
+*   Pull the latest version of `main` and install all dependencies with `renv::restore()`.
+*   Create a Git branch for your pull request (PR). You could use `usethis::pr_init("brief-description-of-change")`.
+*   Make your changes, commit to git, and then create a PR by running `usethis::pr_push()`, and following the prompts in your browser.
```

Comment:
Would want to make it clear that `usethis` is optional and people can still use any other methods to create PRs.